src/components/Map.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import moment from 'moment-timezone';
+
+import Map from './Map';
+
+interface Element {
+  getBoundingClientRect: () => {
+    width: number;
+    height: number;
+    top: number;
+    left: number;
+    bottom: number;
+    right: number;
+  };
+}
+
+describe('<Map />', () => {
+  const mockState = {
+    meetings: {
+      'foo': {
+        isInPerson: true,
+        latitude: 40.712776,
+        longitude: -74.005974,
+        name: 'First Meeting',
+        start: moment(),
+      },
+    },
+  };
+
+  //save copy of original
+  const originalGetBoundingClientRect = Element.prototype.getBoundingClientRect;
+
+  //override getboundingclientrect
+  beforeAll(() => {
+    Element.prototype.getBoundingClientRect = jest.fn(
+      () =>
+        ({
+          width: 120,
+          height: 120,
+          top: 0,
+          left: 0,
+          bottom: 0,
+          right: 0,
+        } as DOMRect)
+    );
+  });
+
+  //restore original
+  afterAll(() => {
+    Element.prototype.getBoundingClientRect = originalGetBoundingClientRect;
+  });
+
+  it('renders with one meeting', () => {
+    const { container } = render(
+      <Map
+        filteredSlugs={Object.keys(mockState.meetings)}
+        listMeetingsInPopup={false}
+        state={mockState}
+        setState={jest.fn()}
+        mapbox="pk.123456"
+      />
+    );
+    expect(container).toBeTruthy();
+  });
+
+  it('renders with multiple meetings', () => {
+    const mockStateMultiple = {
+      ...mockState,
+      meetings: {
+        ...mockState.meetings,
+        'bar': {
+          isInPerson: true,
+          latitude: 34.052235,
+          longitude: -118.243683,
+          name: 'Second Meeting',
+          start: moment(),
+        },
+      },
+    };
+    const { container } = render(
+      <Map
+        filteredSlugs={Object.keys(mockStateMultiple.meetings)}
+        listMeetingsInPopup={true}
+        state={mockStateMultiple}
+        setState={jest.fn()}
+        mapbox="pk.123456"
+      />
+    );
+    expect(container).toBeTruthy();
+
+    /*todo wait for map to load
```

Comment:
The usual pattern I've seen for waiting for an element is to use `waitFor` with `queryBy*` instead of `getBy*`, but if there's strong potential to switch libs I'm down to hold off :)